pkg/apiserver/installer/cmhandlers.go

```diff
@@ -106,7 +107,7 @@ func (ch *CMHandlers) registerResourceHandlers(a *MetricsAPIInstaller, ws *restf
 	}
 
 	mediaTypes, streamMediaTypes := negotiation.MediaTypesForSerializer(a.group.Serializer)
-	allMediaTypes := append(mediaTypes, streamMediaTypes...)
+	allMediaTypes := append(mediaTypes, streamMediaTypes...) //nolint: gocritic
```

Comment:
it feels like we could already do `ws.Produces(mediaTypes..., streamMediaTypes...)` below instead of creating a new slice no?

Reply:
As far as I know, this is not valid Go :-(

Cf https://go.dev/play/p/Q6jpRq0pNcQ

Of course, I could refactor this code to comply with gocritic, but I feel it's a valid false positive...

Reply:
Sounds good I will trust your feelings on this one :+1: